runtime.js

```diff
@@ -460,10 +460,12 @@
 
       // Pre-initialize at least 20 temporary variables to enable hidden
       // class optimizations for simple generators.
```

Comment:
Ah yes, while it's true that temp variable names are generated in depth-first AST order, code execution is likely to be non-linear. This makes me think the original approach was always slightly broken if there were more than 20 temp variables…

How about something like this?
```js
for (var name in this) {
  // Not sure about the optimal order of these conditions:
  if (name.charAt(0) === "t" &&
      hasOwn.call(this, name) &&
      !isNaN(+name.slice(1)) {
    this[name] = undefined;
  }
}
```